style.css

```
@@ -0,0 +1,689 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+  text-rendering: optimizeLegibility;
+  text-decoration: none;
+}
+
+body {
+  margin: 0;
+  font-family: "Montserrat";
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  border: 0;
+}
+
+.section {
+  padding-top: 41px;
+  padding-bottom: 20px;
+}
+
+.section:nth-child(2n) {
+  background-color: #edf7fa;
+}
+
+.section:nth-child(2n + 1) {
+  background-color: #fff;
+}
+
+.container {
+  width: 375px;
+  margin: 0 auto;
+}
+
+/*----- Hero header -------*/
+
+.header {
+  position: relative;
+  width: 100vw;
+  height: 520px;
+}
+
+.hero-img {
+  background-image: url(./assets/frontEnd.avif);
+  background-size: cover;
+  height: 520px;
+  width: 100vw;
+  background-position: 5%;
+  z-index: -1;
+  overflow: hidden;
+}
+
+.header-links {
+  position: absolute;
+  top: 9px;
+  right: 50px;
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  filter: drop-shadow(3px 3px 2px rgba(0, 0, 0, 0.1));
+}
+
+.header-links img {
+  height: 40px;
+}
+
+.header-text {
+  position: absolute;
+  top: 121px;
+  left: 30px;
+  right: 131px;
+  bottom: 197px;
+  color: white;
+  max-width: 250px;
+}
+
+.header-text p {
+  font-size: 13px;
+  line-height: 18px;
+}
+
+.header-text h1 {
+  font-size: 40px;
+  margin: 0;
+}
+
+.header-text p span {
+  font-weight: 700;
+}
+
+.header-text h2 {
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.header-pic {
+  position: absolute;
+  top: 336px;
+  right: 24px;
+  border-radius: 50%;
+  border: 3px solid white;
+}
+
+/*--------- Introduction section----------*/
+
+.intro-section {
+  font-size: 17px;
+  line-height: 25px;
+  margin: 45px 24px 43px 24px;
+}
+
+.intro-section img {
+  padding-right: 10px;
+}
+
+/*--------- Skills Section----------------*/
+
+.tech-section h3 {
+  font-size: 22px;
+  text-align: center;
+  color: white;
+  background-color: red;
+  margin: 26px 150px;
+}
+
+.tech-section p {
+  font-size: 17px;
+  line-height: 29px;
+  text-align: center;
+  margin: 33px 24px;
+}
+
+/*---------Projects----------------------*/
+
+.featured-projects {
+  font-size: 22px;
+  text-align: center;
+  color: white;
+  background-color: red;
+  margin: 0px 61px;
+}
+
+.project-image-group {
+  position: relative;
+}
+
+.project-image {
+  margin: 40px 24px 9px 24px;
+}
+
+
+.project-overlay {
+  position: absolute;
+  top: 0;
+  margin: 40px 24px 9px 24px;
+} 
```

I think you did not add the background for the overlay yet, maybe you did save that for the future. It might work with the margins, otherwise you could add the position to 0 in all directions to get it over the image. I did like this:
```suggestion
position: absolute;
    top: 0;
    right: 0;
    bottom: 0;
    left: 0;
    background: rgba(1, 61, 120, 0.2)
```